subprojects/docs/src/docs/userguide/jvm/building_java_projects.adoc

```diff
@@ -224,21 +224,36 @@ include::sample[dir="snippets/java/basic/kotlin",files="build.gradle.kts[tags=ja
 
 That's also how you can change the verbosity of the compiler, disable debug output in the byte code and configure where the compiler can find annotation processors.
 
-Two common options for the Java compiler are defined at the project level:
+[[sec:java_cross_compilation]]
+=== Targeting a specific Java version
+
+By default, Gradle will compile Java code to the language level of the JVM running Gradle.
+
+Since version 9, the Java compiler can be configured to produce bytecode for an older Java version, additionally making sure the code does not use any APIs from a more recent version.
```

Comment:
```suggestion
Since version 9, the Java compiler can be configured to produce bytecode for an older Java version while making sure the code does not use any APIs from a more recent version.
```